starfish/pipeline/filter/white_tophat.py

```diff
@@ -1,38 +1,79 @@
 from typing import Optional
 
 import numpy as np
+from scipy.ndimage.filters import maximum_filter, minimum_filter
+from skimage.morphology import disk, ball
 
 from starfish.image import ImageStack
 from ._base import FilterAlgorithmBase
 
 
 class WhiteTophat(FilterAlgorithmBase):
     """
-    Performs "white top hat" filtering of an image to enhance spots. "White top hat filtering" finds spots that are both
-    smaller and brighter than their surroundings.
+    Performs "white top hat" filtering of an image to enhance spots. "White top hat filtering"
+    finds spots that are both smaller and brighter than their surroundings.
 
     See Also
     --------
     https://en.wikipedia.org/wiki/Top-hat_transform
     """
 
-    def __init__(self, disk_size, **kwargs):
-        """Instance of a white top hat morphological masking filter which masks objects larger than `disk_size`
+    def __init__(self, masking_radius: int, is_volume: bool=False, **kwargs) -> None:
```

Comment:
i thought we came up with an alternate term for `is_volume`.

also, `is_volume` is not documented.

Reply:
Updated docstring. We did come up with a new term, which is #301 and will follow.